src/Og.php

```diff
@@ -419,4 +421,27 @@ public static function getSelectionHandler($entity, $bundle, $field_name, array
     return \Drupal::service('plugin.manager.entity_reference_selection')->createInstance('og:default', $options);
   }
 
+  /**
+   * Creates and saves an OG membership.
```

Comment:
this should follow the strictness of `og_membership_create()`:

* validate the group audience field
* Throw exception if membership already exists.